documentation/release-checklist.md

```diff
@@ -19,6 +19,8 @@ OR
   - [ ]  If the release is being cut more than couple of weeks modify [YAML](https://github.com/dotnet/msbuild/tree/main/azure-pipelines/vs-insertion.yml) (and merge to affected MSBuild branches) of the [VS insertion pipeline](https://devdiv.visualstudio.com/DevDiv/_build?definitionId=24295) so that it schedules insertions from MSBuild `vs{{THIS_RELEASE_VERSION}}` to VS `main`. Keep scheduled daily insertions to simplify your workflow and exclude `vs{{THIS_RELEASE_VERSION}}` from triggering insertion on each commit.
 
 ### Branching from main
+- [ ]  Associate the `vs{{THIS_RELEASE_VERSION}}` branch with the next VS {{THIS_RELEASE_VERSION}} release channel \
+`darc add-default-channel  --channel "VS {{THIS_RELEASE_VERSION}}" --branch vs{{THIS_RELEASE_VERSION}} --repo https://github.com/dotnet/msbuild`
```

Comment:
Actually - is it OK to create association that long before branch creation? Arcade checks if the branch to exist:

```
> darc add-default-channel  --channel "VS 17.14" --branch vs17.14 --repo https://github.com/dotnet/msbuild
Warning: Could not find an eng/Version.Details.xml at 'https://github.com/dotnet/msbuild@vs17.14'. Dependency updates may not happen as expected.
Continue? (y/n)
```

Not sure if it might happen to be a requirement going forward?

Let's keep it as part of the branching of current release then?

Reply:
Yeah it should be fine to do it before the branch exists, AFAIK that's just a warning.

Reply:
Ok - applied to the suggested changes